contribs/docker/Dockerfile-db

```diff
@@ -4,7 +4,8 @@ MAINTAINER Wazo Maintainers <dev@wazo.community>
 ADD . /usr/src/wazo-webhookd
 WORKDIR /usr/src/wazo-webhookd
 RUN true \
-    && python3 setup.py install \
+    && python3 -mpip install -U pip \
+    && pip install --no-cache-dir -c constraint-stretch.txt -c constraint-tarballs.txt -e . \
```

Comment:
`python3 setup.py install` also installs dependencies but with `easy_install` instead of `pip`. And `easy install` doesn't support constraint files.

Before this change, the python package was not having any dependencies, that why `python3 setup.py install` was not installing them.

A python package should be able to be installed with all its depends with just "pip install <url/path_to_source_or_zip>".

In (not so) old python world, people was doing this in `setup.py` do mix install_requires and requirements files:

```
with open('requirements.txt') as f:
    required = f.read().splitlines()

setup(
    ....
    install_requires=required,
)
```

It's easy for people to list all `flavor` (optional deps) of a package (here `test` or `integration`), than guessing which requirements file it should use before having a working environment.

Reply:
That's said, in this special case, I can test if the container build with `--no-deps`.

Reply:
> That's said, in this special case, I can test if the container build with `--no-deps`.

I tested it, some deps are missing in the db base image.

Reply:
ok no problems, I now understand